express-post-json/index.js

```diff
@@ -20,8 +20,8 @@ app.post('/api/grades', (req, res) => {
   gradeObject.id = nextId;
   grades[nextId] = gradeObject;
   nextId++;
-  res.json(grades);
   res.status(201);
+  res.json(grades);
```

Comment:
Ok, good and like in the GIF, only the one object of the created grade should be in the response body.

Reply:
ok, I done